h2o-core/src/main/java/hex/Model.java

```diff
@@ -1417,6 +1452,25 @@ public double lift_top_group() {
     );
   }
 
+  protected AdaptFrameParameters makeAdaptFrameParameters() {
+    return _parms;
+  }
+
+  public interface AdaptFrameParameters {
+    Parameters.CategoricalEncodingScheme getCategoricalEncoding();
+    String getWeightsColumn();
+    String getOffsetColumn();
+    String getFoldColumn();
+    String getResponseColumn();
+    double missingColumnsType();
+    int getMaxCategoricalLevels();
+    default String[] getNonPredictors() {
```

Comment:
as you're touching this and making it part of an interface, do you think it makes sense renaming this method at the same time according to https://h2oai.atlassian.net/browse/PUBDEV-8078 ?

Reply:
@sebhrusen good point - I was going to do it but then I realized this is going on the fix release branch and such API change shouldn't be there, I will do it in a follow-up PR against master, sounds okay?